testing

mumblejumble opened the pull request.

mumblejumble committed:
not sure

mumblejumble committed:
testing, not final

mumblejumble committed:
testing not final

mumblejumble committed:
total_guests = 0
    total_tables = 0
    for guests in families:
        total_guests = total_guests + len(guests)
        total_tables = total_tables + (-((-len(guests)) // table_size))
    return total_guests, total_tables

mumblejumble committed:
not final

mumblejumble commented:
Now I am task_02, stocked on how to convert the loop into list like this:
['Dear Jen,\nI look forward to meeting with you on 2015.\nBest,\nMe',
'Dear Max,\nI look forward to meeting you on March 3.\nBest\nMe']

my code gives me 
Dear Jen,
I look forward to meeting with you on 2015.
Best,
Me
Dear Max,
I look forward to meeting with you on March 3.
Best,
Me

which looped successfully. 
I need a hint now. 
i am guessing im missing some steps for converting printout to list

jwokaty commented:
@mumblejumble Create an empty list. Add your formatted statements each time you loop to your new list.

mumblejumble committed:
for appoint in appointments:
        f_email = statement.format(appoint[0], appoint[1])
        print f_email
        newlist = [f_email, f_email]
    return newlist

mumblejumble commented:
This is interesting. now my new code gives me
['Dear Max,\nI look forward to meeting with you on March 3.\nBest,\nMe', 'Dear Max,\nI look forward to meeting with you on March 3.\nBest,\nMe']
It did make the returned a list. However, it only looped list index 1 which is ('Max', 'March 3')
list index 0 got omitted?
But then when i printed f_email, whole list was looped.

mumblejumble committed:
used append

jwokaty commented:
@mumblejumble Your code doesn't add a value to the list. It just replaces the existing value. Take a look at https://docs.python.org/2/tutorial/datastructures.html or the course materials to find out how to add items to lists.

mumblejumble committed:
args corrected

mumblejumble committed:
args corrected
